### ISSUE TITLE
mosquitto sends Online as LWT if broker and client died but only the server was restarted

### ISSUE THREAD

**Issue by justcsdr** [eclipse/mosquitto#1165]
If the broker dies when the client is online so that LTW is Online and the client dies too and only the broker is restarted it will send Online as LWT even that the client never returns.

**Comment by ralight** [comment]
I'm not entirely sure I follow, it sounds like you're saying that if a client uses a LWT of `Online`, then the broker will publish that LWT if the broker dies and the client doesn't return. Is that correct?

**Comment by justcsdr** [comment]
Yes. Let's say that the power goes off when the client was online and when the power comes on, the client doesn't restart (fried electronics or something), the broker will say that the client is online as it was the last time when the broker knew something about the client.
